### ISSUE TITLE
Remove zlib from wx widgets

### ISSUE THREAD

**Issue by project64** [project64/project64#673]
there are two copies of zlib (Source\3rd Party\zlib) and (3rd Party\wx\src\zlib) we do not need two copies 

remove the one from wx widgets

**Comment by JunielKatarn** [comment]
As a note, wx\src\zlib breaks the build using 'Debug' configuration.

**Comment by JunielKatarn** [comment]
Update: This issue looks exclusive to VC15 (again, only Debug configuration).
```
fatal error C1189: #error:  Macro definition of vsnprintf conflicts with Standard Library func
tion declaration [Source\3rd Party\wx\wx_wxzlib.vcxproj]
```
I'll look into it.

**Comment by JunielKatarn** [comment]
I gave it a try yesterday, but it looks like the code on wx_zlib is necessary for wx_png, which is in turn necessary for Glide64.

On another note, this PR fixes the build issue with MSVC2010+ (Debug). #682

**Comment by JunielKatarn** [comment]
I updated the wx/zlib source to fix build issues. (Still unable to delete it from the code base).
#687

**Comment by project64** [comment]
I have not looked at it, it would require wx_png to use the other zlib or we are looking at removing zlib and getting the project to use wx_zlib instead.

**Comment by oddMLan** [comment]
fixed with the removal of wxWidgets 68a8ed03696f3365430d8cca14d5084100165cd0
